programmers/난이도별/level02/최대값과_최솟값/HyeonJeong.py

```diff
@@ -0,0 +1,9 @@
+def solution(s):
+    list = [int(i) for i in s.split(' ')]
+    return str(min(list)) + ' ' + str(max(list))
```

Comment:
```suggestion
    return f'{min(list)} {max(list)}'
```
이렇게 표현할 수 있습니다.

![image](https://user-images.githubusercontent.com/18749057/92763178-3e2a6600-f3ce-11ea-8761-181a495176d3.png)


관련링크: https://docs.python.org/ko/3/tutorial/inputoutput.html